src/javascript/app/pages/user/set_currency.js

```diff
@@ -59,56 +60,74 @@ const SetCurrency = (() => {
             $('#set_currency, .select_currency').setVisibility(1);
 
             const $currency_list = $('.currency_list');
+            const popup_selector = '#set_currency_popup_container';
             $('.currency_wrapper').on('click', function () {
                 $currency_list.find('> div').removeClass('selected');
                 $(this).addClass('selected');
-            });
+                const $popup_container = $('#set_currency_popup');
+                const $popup_content   = $('#set_currency_popup_content');
 
-            const $form  = $('#frm_set_currency');
-            const $error = $form.find('.error-msg');
-            $form.on('submit', (evt) => {
-                evt.preventDefault();
-                $error.setVisibility(0);
-                const $selected_currency = $currency_list.find('.selected');
-                if ($selected_currency.length) {
-                    BinarySocket.send({ set_account_currency: $selected_currency.attr('id') }).then((response_c) => {
-                        if (response_c.error) {
-                            $error.text(response_c.error.message).setVisibility(1);
-                        } else {
-                            localStorage.removeItem('is_new_account');
-                            Client.set('currency', response_c.echo_req.set_account_currency);
-                            BinarySocket.send({ balance: 1 });
-                            BinarySocket.send({ payout_currencies: 1 }, { forced: true });
-                            Header.displayAccountStatus();
+                let localized_text = '';
+                if (isCryptocurrency($(this).attr('id'))) {
+                    localized_text = localize('You have chosen <strong>[_1]</strong> as the currency for this account. You cannot change this later. You can have more than one cryptocurrency account.', $(this).attr('id'));
```

Comment:
it's better to not include the element `<strong/>` in translation string, could you move it to the second params instead: `<strong>$(this).attr('id')</strong>` ?